tests/tasks/setup/kitctl/setup-control-plane.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: control-plane-setup
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/version: "0.2"
```

Comment:
Why is this label needed?

Reply:
You are right we don't need these labels yet. Those are just to help us identify the version of the tasks. I will remove those.